lib/yaaf/form.rb

```diff
@@ -60,5 +66,10 @@ def handle_transaction_rollback(exception)
       run_callbacks :rollback
       raise exception
     end
+
+    def rails_version_less_than_6_1?
+      ActiveModel::VERSION::MAJOR < 6 ||
+        ActiveModel::VERSION::MAJOR == 6 && ActiveModel::VERSION::MINOR.zero?
```

Comment:
We can use `Gem::Version` to check this if I'm not mistaken

```suggestion
      Gem::Version.new(ActiveModel::VERSION) < Gem::Version.new('6.1')
```

I wonder if `ActiveModel::VERSION` it's not already a `Gem::Version` object

Reply:
@brunvez I merged it but it broke Rails 4 tests, so I'm gonna revert it